src/GafferUI/AnimationGadget.cpp

```diff
@@ -767,7 +767,18 @@ void AnimationGadget::plugDirtied( Gaffer::Plug *plug )
 
 std::string AnimationGadget::getToolTip( const IECore::LineSegment3f &line ) const
 {
-	if( const Animation::ConstKeyPtr key = keyAt( line ) )
+	std::pair< Gaffer::Animation::ConstKeyPtr, Gaffer::Animation::Direction > keyTangent = tangentAt( line );
+	if( keyTangent.first )
+	{
+		const Gaffer::Animation::Tangent& tangent = keyTangent.first->tangent( keyTangent.second );
+		std::ostringstream os;
+		os.precision( 4 );
+		os << "Direction: " << Gaffer::Animation::toString( tangent.direction() );
+		os << "<br>Slope: " << tangent.getSlope();
+		os << "<br>Scale: " << tangent.getScale();
```

Comment:
I hadn't really thought about these being user-facing - the way scale is mapped doesn't really feel like something artists are going to be able to understand, but I guess it doesn't really make sense to present it differently here.

It might at least make sense to skip showing it on segments using interpolators that aren't using scale?